src/super_gradients/recipes/cityscapes_stdc_base.yaml

```diff
@@ -1,7 +1,7 @@
 # STDC Base training params
 
 defaults:
-  - training_hyperparams: cityscapes_stdc_train_params
+  - training_hyperparams: cityscapes_train_params
```

Comment:
Lets rename them to cityscapes_default_train_params

Reply:
Done